Source/Core/Core/HW/WiimoteReal/WiimoteReal.h

```diff
@@ -88,6 +88,9 @@ friend class WiimoteEmu::Wiimote;
 	virtual int IORead(u8* buf) = 0;
 	virtual int IOWrite(u8 const* buf, size_t len) = 0;
 	virtual void IOWakeup() = 0;
+#ifdef __APPLE__
+	virtual void IOCancel() = 0;
```

Comment:
This can just be made available to all implementations; facilitating removal of platform independent ifdefs.